corehq/apps/hqwebapp/static/app_manager/less/form_editing.less

```diff
@@ -36,12 +36,16 @@
 }
 
 .btn-form-action {
+  border: 1px solid white;
+  padding: 5px 10px;
+  margin-right: 10px;
   color: white;
   .transition(.5s background);
 
   &:hover {
     color: white;
     background-color: @cc-brand-low;
+    border-color: #999;
```

Comment:
what is this color? can we refer to a named color in the palette?

Since we're in the bootstrap 3 stylesheet:
https://www.commcarehq.org/styleguide/atoms/#atoms-colors